popgen/stationarity.go

```diff
@@ -53,24 +53,23 @@ func MultiFaToAFS(aln []*fasta.Fasta) AFS {
 
 //GvcfToAFS reads in a Gvcf file, parses the genotype information, and constructs an AFS struct.
 //TODO: This function will change when we update the gVCF stuff.
-func GVCFToAFS(filename string) AFS {
+func VcfToAFS(filename string) AFS {
 	var answer AFS
 	answer.sites = make([]*SegSite, 0)
-	alpha := vcf.GoReadGVcf(filename)
+	alpha, _ := vcf.GoReadToChan(filename)
 	var currentSeg *SegSite
 	var j int
-	for i := range alpha.Vcfs {
+	for i := range alpha {
 		currentSeg = &SegSite{i: 0, n: 0}
 		//gVCF converts the alt and ref to []DNA.base, so structural variants with <CN0> notation will fail to convert. This check allows us to ignore these cases.
-		if !strings.ContainsAny(i.Alt, "<>") {
-			g := vcf.VcfToGvcf(i)
-			for j = 0; j < len(g.Genotypes); j++ {
-				if g.Genotypes[j].AlleleOne != -1 && g.Genotypes[j].AlleleTwo != -1 { //check data for both alleles exist for sample.
+		if !strings.ContainsAny(i.Alt[0], "<>") {//By definition, segregting sites are biallelic, so we only check the first entry in Alt.
+			for j = 0; j < len(i.Samples); j++ {
+				if i.Samples[j].AlleleOne != -1 && i.Samples[j].AlleleTwo != -1 { //check data for both alleles exist for sample.
```

Comment:
This is the second time I am seeing -1 come up.  I am thinking it means "empty," but I am not sure if it is an "allowable" value, or if it is an error.